app/views/admin/creator_settings/_form.html.erb

```diff
@@ -87,15 +87,15 @@
     <legend class="crayons-field__label">Finally, please agree to the following:</legend>
     <div>
       <p class="crayons-field__description">You will have the opportunity to establish the Code of Conduct and Terms and Conditions for the users of your Forem during the setup process.</p>
-      <div class="mb-2">
+      <div class="mb-2" data-controller="modal" data-modal-root-selector-value="#creator-settings-modal-root" data-modal-content-selector-value="#accept-code-of-conduct-modal" data-modal-title-value="Code of Conduct" data-modal-size-value="m">
         <%= hidden_field_tag :checked_code_of_conduct, "0" %>
         <%= check_box_tag :checked_code_of_conduct, "1", false, class: "crayons-checkbox", required: true %>
-        <label for="checked_code_of_conduct">I agree to uphold our <a href="/code-of-conduct">Code of Conduct</a>.</label>
+        <label for="checked_code_of_conduct">I agree to uphold our <a data-action="modal#toggleModal">Code of Conduct</a>.</label>
```

Comment:
(refactor / blocking) - because these links don't have an `href`, they're not keyboard focusable. I know we don't want to actually route users to the `/code-of-conduct` path, but I think we will need to suppress this in JS using `event.preventDefault()` to ensure keyboard/screen reader users can open these modals

Reply:
I spoke with @nickytonline today and after some pairing on the best way forward to fix these inaccessible links, we decided that it would probably be best to revert these modal changes and instead keep these links as links, while ensuring that they open the COC and TOS as expected in a new window (so that we do not disturb a Creator's onboarding flow). I've proposed this solution to the Admin Experience Pod [here](https://forem-team.slack.com/archives/C01S4QZUDNC/p1638394459209700) and have opened a draft PR for the proposed solution [here](https://github.com/forem/forem/pull/15634).

Please let me know if we're overlooking something, otherwise, I plan on closing this PR in favor of the new implementation once I get the team's and your sign off to do so, @aitchiss!